pages/user/user.wxml

```diff
@@ -14,16 +17,16 @@
   <view class="section__events">
 
     <block wx:for="{{events}}" wx:for-item="event">
-      <navigator url="/pages/event/event?id={{ event.id }}">
+      <navigator url="/pages/event/event">
```

Comment:
obviously this line breaks existing feature. please always review your own PR first. @royyirao2017